README.md

```diff
@@ -1,25 +1,25 @@
-), which the company boasts to be "the industry's largest in cloud computing." 
+), which the company says is "the industry's largest in cloud computing." 
 
-As Microsoft workers, we've been made complicit.
+As Microsoft workers, whilst we are proud of the efforts the company has made towards improving sustainability, we must do more.
 
 The Global Climate Strike is an opportunity for Microsoft to set an example as a leader in the tech industry. The company has taken many initiatives that focus on individual contributions, such as educating employees on composting, reducing waste, and making our [campus carbon neutral](https://blogs.microsoft.com/on-the-issues/2017/07/14/creating-carbon-free-headquarters-puget-sound-region/ ). There are also efforts to leverage artificial intelligence to solve specific and important climate-related problems. However, given the urgency and complexity of the crisis, these actions are not enough.
 
-If we want to make real impact, all of us need to mobilize, work together, and demand a fundamental change in Microsoft's priorities. It's clear that the tech industry is one of the main culprits behind our burning planet, both in consuming a great deal of fossil fuels through power hungry datacenters and supporting Big Oil companies. But we also have to recognize that we're equipped with so many resources that we can leverage to steer our industry and government in the right direction. 
+If we want to make real impact, all of us need to mobilize, work together, and demand a fundamental change in Microsoft's priorities. 
 
-Following our mission to "Empower every person and every organization in the planet to achieve more", it is essential that we secure a planet with livable conditions for *everyone*. This includes victims of [disaster displacement](https://weather.com/forecast-change/news/2019-06-04-how-the-middle-east-drought-and-flood-cycle-will-probably-lead-to ) often from the Global South and Black and Brown communities who are [disproportionately affected](https://www.ebony.com/news/environmental-justice-begins-with-understanding-how-climate-change-impacts-african-american-communities/) by climate change.
+Following our mission to "Empower every person and every organization in the planet to achieve more", it is essential that we secure a planet with livable conditions for *everyone*. 
```

Comment:
This part particularly should be left as is. Microsoft tends to be more affluent workers, and I think it's very important to center marginalized people. I think it would be particularly tone-deaf to remove this.

Reply:
I can see that perspective, because I agree there's a serious problem with social and inequitable impacts from climate change. For example, Bangladesh and the Pacific Islands will be flooded from ocean levels rises, creating _millions_ of climate refugees. 
I wish the original author had engaged the Microsoft community in advance of this presumptive post, so some consensus could have been reached on what strength of language people agree with.

Reply:
I agree with @ealexjordan

Reply:
@girifox it looks like this was co-authored by multiple Microsoft workers. Who are you to claim they didn’t “engage the Microsoft community” just because you weren’t part of it?

Reply:
I'm engaged with the worldwide sustainability community, our largest internal climate interest group, and there was no approach or invitation to that group to review a draft. Therefore, the author/s were not inclusive of the whole community, and can't claim to speak for the whole. (Hey I'd be fine if there was a caveat on their post about its voice being from a _subset_ of interested employees. I'm freaked-out about climate change. I care, so I think our energy would be way better invested outward towards positive change, and not bickering about a lack of inclusive on a post like this.)

Reply:
This letter doesn't speak for the whole, but rather to the whole. The internal climate interest group is free to listen to it or not, to engage with it or not, and to write its own letter or not. These are two different groups taking two different approaches. This is good! The best activism efforts succeed when there are a diversity of tactics employed.

What gets in the way is when groups start calling out the other's tactics for not conforming to their own views on what is the best way. Right now, this is what you're doing.

> I care, so I think our energy would be way better invested outward towards positive change, and not bickering about a lack of inclusive on a post like this.

To be clear, you're the one bickering here and trying to get things changed. If you think your energy is better spent elsewhere, then why are you bickering here?